docs/user_docs/installation/install-with-helm/install-kubeblocks-with-helm.md

```diff
@@ -57,28 +59,50 @@ helm install kubeblocks kubeblocks/kubeblocks \
 ````
 
 If you want to install KubeBlocks with custom tolerations, you can use the following command:
+
 ```bash
 helm install kubeblocks kubeblocks/kubeblocks \
     --namespace kb-system --create-namespace \
     --set-json 'tolerations=[ { "key": "control-plane-taint", "operator": "Equal", "effect": "NoSchedule", "value": "true" } ]' \
     --set-json 'dataPlane.tolerations=[{ "key": "data-plane-taint", "operator": "Equal", "effect": "NoSchedule", "value": "true" } ]'
 ```
 
+If you want to install KubeBlocks with a specified version, follow the steps below.
+
+1. View the available versions in [KubeBlocks Release](https://github.com/apecloud/kubeblocks/releases/).
+2. Specify a version with `--version` and run the command below.
+
+   ```bash
+   helm install kubeblocks kubeblocks/kubeblocks \
+    --namespace kb-system --create-namespace --version="x.x.x"
+   ```
+
+  :::note
+
+  By default, kbcli installs KubeBlocks with the same version. If you want to install KubeBlocks with a different version, ensure that the minor versions of kbcli and KubeBlocks match.
```

Comment:
should it be 'major' versioin instead of 'minor' version?

Reply:
> should it be 'major' versioin instead of 'minor' version?

Fixed. Plz review docs again. THX!